CMakeLists.txt

```diff
@@ -168,6 +168,7 @@ find_package( breakpad QUIET )
 find_path(LINUXMOUNT NAMES "linux/mount.h")
 if(NOT LINUXMOUNT)
```

Comment:
I don't think the test for `linux/mount.h` is sufficient to test if `open_tree` syscall is supported.  It may have to be done at runtime using `uname` or maybe as an explicit build flag ?

FWIW - I found the `linux/mount.h` was available in toolchains built for platforms running 4.9 kernels, and if built and run on one of those platforms I get the following error when trying to add a mount:
```
ERR: open_tree failed errno: 38 container: 734
```